en/orm/table-objects.rst

```diff
@@ -166,17 +166,17 @@ buildRules
 
 .. php:method:: buildRules(Event $event, RulesChecker $rules)
 
-The ``Model.buildRules`` event is fired before after a rules instance has been
-created and the table's ``beforeRules()`` method has been called.
+The ``Model.buildRules`` event is fired after a rules instance has been
+created and before the table's ``beforeRules()`` method has been called.
```

Comment:
I'll have to double-check the code, but I thought the order was buildRules, beforeRules, then afterRules.  Thanks!